CHANGES.rst

```diff
@@ -1,5 +1,8 @@
 0.4.0 (unreleased)
 =======
+general
+-------
+- Added documentation for dark current reference files. [#193]
```

Comment:
```suggestion
- Added documentation for dark current reference files. [#232]
```

Reply:
We add the PR number so it links the log to the actual changes.

Reply:
Changed.. I (incorrectly) used the issue number.